<?xml version="1.0" encoding="UTF-8"?>
<module external.linked.project.id=":01._Wrapper_classes-Identify_a_possible_exception:main" external.linked.project.path="$MODULE_DIR$/../../../../01. Wrapper classes/Identify a possible exception" external.root.project.path="$MODULE_DIR$/../../../.." external.system.id="GRADLE" external.system.module.group="11._Wrapper_classes" external.system.module.type="sourceSet" external.system.module.version="unspecified" type="JAVA_MODULE" version="4">
  <component name="NewModuleRootManager">
    <output url="file://$MODULE_DIR$/../../../../01. Wrapper classes/Identify a possible exception/out/production/classes" />
    <exclude-output />
    <content url="file://$MODULE_DIR$/../../../../01. Wrapper classes/Identify a possible exception/src">
      <sourceFolder url="file://$MODULE_DIR$/../../../../01. Wrapper classes/Identify a possible exception/src" isTestSource="false" />
    </content>
    <orderEntry type="inheritedJdk" />
    <orderEntry type="sourceFolder" forTests="false" />
    <orderEntry type="module" module-name="11._Wrapper_classes.util.main" />
  </component>
</module>